doc/source/examples/sphinx_examples/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Is this file required?

Reply:
I think yes. And we dont need sphinx_design.rst https://github.com/ansys/ansys-sphinx-theme/pull/191#discussion_r1105314511 instead
I was trying to download all the txt files from the sphinx design and download to this directory, so that it will be useful for

1. get the file name from txt name.
2. get the raw directive and rendered docs from the txt files. (useful as a quick reference.)as shown in the SS.
3. Exclude the files if it is showing some error while rendering. 
![sphinx-design](https://user-images.githubusercontent.com/104772255/218685716-31686ca6-5566-4c1a-9e99-009427eac0e4.PNG)
But i am open to changes , to keep all the content of text file in single rst.